airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/InitialSyncCtidIterator.java

```diff
@@ -330,4 +353,14 @@ public void close() throws Exception {
     }
   }
 
+  private boolean isCdcSync(CtidStateManager initialLoadStateManager) {
```

Comment:
interesting... I wonder if there is a way to avoid replicating this function implementation for each db connector...

Reply:
Theoretically yes, but I'm hoping at this point the new CDK handles this. 

I think that any solution or common code we have now will include passing in this logic iin some capacity as the `CtidGlobalStateManager`, `MySqlInitialLoadGlobalStateManager` and `MySqlInitialLoadGlobalStateManager` are different objects even though they could be unified